sys/kern/subr_intr.c

```diff
@@ -1552,14 +1595,18 @@ dosoftints(void)
 void
 intr_pic_init_secondary(void)
 {
-
 	/*
-	 * QQQ: Only root PIC is aware of other CPUs ???
+	 * QQQ: Only root PICs are aware of other CPUs ???
 	 */
-	KASSERT(intr_irq_root_dev != NULL, ("%s: no root attached", __func__));
-
+	uint32_t rootnum;
+	device_t dev;
 	//mtx_lock(&isrc_table_lock);
-	PIC_INIT_SECONDARY(intr_irq_root_dev);
+	for (rootnum = 0; rootnum < INTR_ROOT_NUM; rootnum++) {
+		dev = intr_irq_roots[rootnum].dev;
+		if (dev) {
+			PIC_INIT_SECONDARY(dev, rootnum);
+		}
```

Comment:
I believe FreeBSD style allows omitting the braces.  Style also needs `dev != NULL`.

Reply:
I fixed the `NULL` check. Since the style guide doesn't _require_ omitting the braces I'd prefer to leave them in out of an abundance of caution for when someone needs to change this.